### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by trbehera**

### CODE REVIEW

**Review by KiranSukhavasi — DISMISSED** [review]

File: docs/installation.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ This document can be used as a quick start guide to setup the development enviro
...
 | Component | Recommended |
 |------- |------|
-| Operating System | Ubuntu 18.04 / Windows 10 |
-| Docker Engine | 18.09 |
-| Docker-compose | 1.21.2 |
-| maven | 3.5.4 |
+| Operating System | Ubuntu\* 18.04 / Windows\* 10 |
+| Docker* Engine | 18.09 |
+| Docker* Compose | 1.21.2 |
+| Maven* | 3.5.4 |
```
Comment (TomBarnes) [inline docs/installation.md:13]: If the name of the tool is "docker-compose", then I think it should be shown that way.  This is not a case of using Docker without attribution.

File: docs/index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Secure Device Onboard is a flexible software solution that simplifies and automa
...
+During deployment of a device enabled with the Secure Device Onboard solution, all the installer needs to do is to add power to the device and connect it to the Internet. The device automatically and rapidly connects to the IoT platform in a secure and trusted manner. Secure Device Onboard combines embedded hardware security credentials with a chain-of-ownership security credential and Rendezvous service to secure the onboarding process and broker the connection to the authorized device management service provider. Today Secure Device Onboard supports Enhanced Privacy ID (EPID) and Elliptic Curve Digital Signature Algorithm-based embedded hardware security credentials.
```
Comment (TomBarnes) [inline docs/index.md]: Not sure why Intel<sup>®</sup> was removed - Enhanced Privacy ID is an intel trademark, not part of open source SDO. Same for EPID.

File: docs/installation.md [hunk 3]
```diff
@@ -94,8 +94,8 @@ sudo systemctl restart docker
 sudo systemctl show --property Environment docker
 ```
 
-## Docker-compose Installation
-To install a specific version of docker-compose (for example **_1.21.2_**) follow these steps:
```
Comment (TomBarnes) [inline docs/installation.md:98]: Same comment here - docker-compose is the name of the tool, not a stand-alone mention of docker.

File: docs/iot-platform-sdk/introduction.md [hunk 4]
```diff
@@ -52,7 +52,7 @@ This component is responsible for the actual onboarding of a device by running T
 
 **restimpl:** This module makes use of 'libops' module to execute TO2 for multiple SDO clients. It configures the 'libops' and provides implementations of a few interfaces that define the way data is provided to TO2 protocol endpoints. It wraps the TO2 protocol implementation of the 'libops' module, by creating separate REST endpoints for managing all incoming TO2 requests. It sends REST calls to OCS to get and store device-specific information, such as ownership voucher, service-info files, TO2 session information among others, then provides these as inputs to the TO2 protocol endpoints.
 
-**epid:** This module contains the Intel<sup>®</sup> Enhanced Privacy ID(Intel<sup>®</sup> EPID)-specific implementations that is used by the 'libops' module to handle Intel<sup>®</sup> EPID-based Secure Device Onboard clients. It contacts the Intel<sup>®</sup> EPID online services to validate Secure Device Onboard clients.
+**epid:** This module contains the Enhanced Privacy ID(EPID)-specific implementations that is used by the 'libops' module to handle EPID-based Secure Device Onboard clients. It contacts the EPID online services to validate Secure Device Onboard clients.
```
Comment (TomBarnes) [inline docs/iot-platform-sdk/introduction.md]: Same comment as above regarding intel trademarks.

File: docs/iot-platform-sdk/running-the-demo.md [hunk 5]
```diff
@@ -297,7 +304,7 @@ For Device Simulation:
...
+Each docker service of IOT Platform SDK has its own configuration file with extension '.env'. When each service starts, the properties stored in the '.env' files will be set as environment variables within the container. The runnable scripts, namely, run-ops, run-ocs, and run-to0scheduler, that are responsible for starting the individual services within the docker container, explicitly maps the set the environment variables to the respective property of the application and passes them as Java* system variables. The properties for the PRI components are configurable using the respective 'application.properties' file. For this simulation and during development, the '.env' files  and the 'application.properties' files are writable. However, in a production environment, change the permissions of the these files to read-only for added security.
```
Comment (TomBarnes) [inline docs/iot-platform-sdk/running-the-demo.md]: Looks thike the right place for "Docker*"

File: docs/installation.md (hunk 1 above)
Comment (trbehera) [reply docs/installation.md:13]: Compose is a Docker tool, the command is docker-compose. 
https://docs.docker.com/compose/

File: docs/index.md (hunk 2 above)
Comment (trbehera) [reply docs/index.md]: I suppose, I will revert this back.

**Review by DukeDavis12 — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Review by TomBarnes — APPROVED** [review]

**Pull request merged**